host-interaction/network/traffic/filter/enumerate-network-filters-via-wfp-api.yml

```diff
@@ -0,0 +1,21 @@
+rule:
+  meta:
+    name: enumerate network filters via WFP API
+    namespace: host-interaction/network/traffic/filter
+    authors:
+      - jakub.jozwiak@mandiant.com
+    scopes:
+      static: function
+      dynamic: call
+    att&ck:
```

Comment:
maybe remove these?

Reply:
Thanks! Updated in https://github.com/mandiant/capa-rules/pull/930/commits/afafccaf8f177cefba9eb4475f294a839ec5ca1a